pkgs/development/misc/vc4/newlib.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, texinfo, flex, bison, fetchFromGitHub, crossLibcStdenv, buildPackages }:
+
+crossLibcStdenv.mkDerivation {
+  name = "newlib";
+  src = fetchFromGitHub {
+    owner = "itszor";
+    repo = "newlib-vc4";
```

Comment:
Perhaps we should merge the "mainline", "msp430", and "vc4" newlib's into a shared derivation to make this a little easier to maintain (presumably you only want one "newlib" at a time). I hadn't realized how many forks of newlib existed! Not necessary for this PR though.